docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+- Grow the community.
+- Maximize the `bus factor`_ of the project.
+- Acknowledge the leadership and time multiple people contribute to our
+  community without demanding more time than any individual can offer.
+  Dividing leadership responsabilies into multiple smaller roles also
+  makes it easier to encourage new people to take on a leadership role
+  without fearing that too much work will be required of them.
+
+This governance structure is a work-in-progress. We welcome both people
+who want to take on a leadership role as well as ideas for our to improve
+this structure.
+
+
+
+Steering Committee and Leadership Roles
+---------------------------------------
+
+Organizational Principles
+`````````````````````````
+The steering committee steers. The goal of the steering committee is to help
+guide the direction of the project. Decisions in the steering committee will 
+focus on how to present project issues to the broader community in a clear way
+rather than making project decisions without community input. 
+
+Issues or pull requests that can be reviewed by any community member can be
+reviewed and approved by the steering committee or steering committee members.
+The steering committee can decide that an issue should be prioritized for wider
+communal discussion or that a pull request requires more discussion or reviews
+than standard before merging. The steering committee can also decide how a
+breaking change (something that changes existing user function calls or program
+outputs) will be presented to the developer and user base for discussion, before
+decisions are made.
+
+Steering committee decisions should strive for consensus and accept the
+majority’s decision. If voting is necessary, it can happen asynchronously and is
+defined as more than half of the members of the steering committee. If a member
+of the steering committee is unable to spend the time necessary to understand
+an issue & vote, they can recuse from a decision so that the number of committee
+members is reduced
+
+Openness is a priority:
+
+- Openness is critical to building trust with the broader community
+- Openness provides a mechanism for non-steering committee members to identify
+  and address steering committee blindspots
+- Openness provides a smoother transition to onboard future steering committee
+  members
+- If steering committee discussions (written or verbal) could include
+  community members without compromising efficiency, they should be invited. The
+  steering committee can schedule discussions without needing to ask about the
+  availability for people outside of the steering committee. If notes from
+  meetings can be openly shared without compromising personal privacy, they
+  should be.
+
+The current members of the tedana steering committee are:
+
+- Name 1 & link to github profile
+- Name 2 & link to github profile
+- Name 3 & link to github profile
+- Name 4 & link to github profile
+- Name 5 & link to github profile
```

Comment:
Is this the maintainer list, or do you see this as a different group ?

Reply:
The steering committee is separate from the maintainers. The maintainer list has no size limit. The steering committee is a smaller group that can more efficiently communicate & meet. I (we?) envisioned these as two separate groups, but several questions above make me wonder if they should be. We've effectively separated the leadership roles from steering committee membership. Does that mean our group of maintainers is effectively the steering committee? If there are ~10 maintainers is that too many people to commit to be involved in all decisions?

Also, one of your comments above mentioned Admin privileges. We don't talk at all about any leader having those privileges or the appropriate situations to use them. That seems like an oversight.

Reply:
Before reading this, I was honestly under the impression that maintainers and steering were the same. I thought we had discussed having representation across the different areas on steering--hence the time spent defining them--but I'm not sure if I had just conflated the two ideas.

Either way, I think a smaller steering makes sense (I'd suggest 6 members). It would also make sense to me to extend that to those folks who regularly come to developer calls, since they're already very familiar with the project and have a history of involvement.

Reply:
Current proposal:
- Leaders can be anyone
- Maintainers are people who have a standing commitment to contribute
- The steering committee is a subset of maintainers
- We still need to clarify the specific things that the steering committee can do that maintainers cannot
It would be good to get @tsalo's thoughts on this.